bokehjs/src/coffee/renderer/guide/categorical_axis.coffee

```diff
@@ -13,6 +9,7 @@ class CategoricalAxis extends Axis.Model
 
   initialize: (attrs, objects) ->
     super(attrs, objects)
+    Collections = require("../../common/base").Collections
```

Comment:
As an abstract (I didn't dig into the code), I believe it's an import order issue.

* Start loading common/base
* Leave common/base while trying to load this file
* This file references common/base, but it hasn't finished loading and Collections isn't defined

Moving this to execute at runtime instead of import time means that `.Collections` has been added to the common/base module.